modular_chomp/code/modules/mob/living/simple_mob/subtypes/slimess/xenobio.dm

```diff
@@ -1,2 +1,238 @@
+	result_amount = 30
+	required = /obj/item/slime_extract/sepia
+
+/decl/chemical_reaction/instant/slime/secertslime
+	name = "Secert Slime "
+	id = "sepia_swapm"
+	result = "change_drug_male"
+	required_reagents = list("water" = 1, "blood" = 1, "phoron" = 3)
+	result_amount = 1
+	required = /obj/item/slime_extract/sepia
+
+/decl/chemical_reaction/instant/slime/secertslime/on_reaction(var/datum/reagents/holder)
+	holder.my_atom.visible_message("<span class='warning'>Infused with the mixture, the core begins to quiver and grow, and soon a new baby slime emerges from it!</span>")
+	new /mob/living/simple_mob/slime/xenobio/redspace(get_turf(holder.my_atom))
+	..()
+
+/decl/chemical_reaction/instant/slime/genderswapf
+	name = "Slime Change F"
+	id = "sepia_swapf"
+	result = "change_drug_female"
+	required_reagents = list("water" = 5)
+	result_amount = 30
+	required = /obj/item/slime_extract/sepia
+
+/decl/chemical_reaction/instant/slime/genderswapm
+	name = "Slime Change M"
+	id = "sepia_swapf"
+	result = "change_drug_male"
+	required_reagents = list("blood" = 5)
+	result_amount = 30
+	required = /obj/item/slime_extract/sepia
+
+/decl/chemical_reaction/instant/slime/sepia_new_slime
+	name = "Slime Spawn"
+	id = "m_sepia_spawn"
+	result = null
+	required_reagents = list("slimejelly" = 5)
+	result_amount = 1
+	required = /obj/item/slime_extract/sepia
+
+/decl/chemical_reaction/instant/slime/sepia_new_slime/on_reaction(var/datum/reagents/holder)
+	holder.my_atom.visible_message("<span class='warning'>Infused with slime jelly, the core begins to quiver and grow, and soon a new baby slime emerges from it!</span>")
+	new /mob/living/simple_mob/slime/xenobio/sepia(get_turf(holder.my_atom))
+	..()
+
+
+
+/mob/living/simple_mob/slime/xenobio/redspace
+	desc = "This slime is rather unknown and strange."
+	color = "#A50021"
+	slime_color = "redspace"
+	coretype = /obj/item/slime_extract/redspace
+	reagent_injected = "mindbreaker"
+
+	mutation_chance = 0
+
+	slime_mutation = list(
+			/mob/living/simple_mob/slime/xenobio/dark,
+			/mob/living/simple_mob/slime/xenobio/redspace,
+			/mob/living/simple_mob/slime/xenobio/bluespace,
+			/mob/living/simple_mob/slime/xenobio/plague
+		)
+
+/obj/item/slime_extract/redspace
+	name = "redspace slime extract"
+	icon_state = "ruby slime extract"
+	description_info = "Who knows what will happen."
+	slime_type = /mob/living/simple_mob/slime/xenobio/redspace
+
+
+/decl/chemical_reaction/instant/slime/redspace_fire
+	name = "Slime Living Fire"
+	id = "m_liquid_fire"
+	result = "neoliquidfire"
+	required_reagents = list("phoron" = 5)
+	result_amount = 30
+	required = /obj/item/slime_extract/redspace
+
+/decl/chemical_reaction/instant/slime/soulstone
+	name = "Slime Soulstone"
+	id = "m_redspace_soulstone"
+	required_reagents = list("blood" = 5)
+	result_amount = 1
+	required = /obj/item/slime_extract/redspace
+
+/decl/chemical_reaction/instant/slime/purple_shrink/on_reaction(var/datum/reagents/holder)

```

Comment:
Should  be `/decl/chemical_reaction/instant/slime/soulstone/on_reaction`?

Reply:
Oh, yeah. Opps.

Reply:
Thank you.

Reply:
You're welcome.
Also, the `/decl/chemical_reaction/instant/slime/virusdish` below too.

Are the dark_neofire and dark_liquidfire WIP? I guessed so from the extract's desc.

Reply:
The extract I am leaving as having minimal because if folks are chill with the idea of the weird, untradtional slimes, and such I will expand it further. If not, I will go back and adjust it.
Those two reactions are what I desire. as of currently.
But the description is more so me nudging players that 'yeah more content will be added to this later'